mantle/kola/README-kola-ext.md

```diff
@@ -62,9 +62,13 @@ Support for rebooting
 
 An important feature of exttests is support for rebooting the host system.
 This allows one to easily test OS updates for example.  To do this, your
-test process should send `SIGTERM` to itself.  For example, in bash use:
+test process should create a stamp file `/run/kola-reboot`, then send
+`SIGTERM` to itself.  For example, in bash use:
```

Comment:
If we support an explicit `/run/kola-reboot`, then the `SIGTERM` "self-signalling" bit is obsolete, right? Why not just let the script `exit 0`?

Reply:
Hmmm.  I don't have a really strong opinion on this but either way is a special case.  Since we need backcompat for a while it seems simpler to keep it as is, but if you or someone else feels strongly we can change it the other way.

Reply:
Right, definitely still need what you have here for backcompat. I mean in addition to that, have the check for `kolaRebootStamp` in the `CLD_EXITED` path.

IMO it's easier to explain to contributors "to reboot, touch this file, then exit", rather than "to reboot, touch this file, signal yourself, then sleep infinity". Being able to just exit also saves more boilerplate in Golang/Rust code where signalling yourself is not just a one-liner. WDYT?

Reply:
Ooh right, the "need to sleep infinity" part is definitely hacky.  OK will rework this.

Reply:
(IOW if it was just `exit 0` versus `kill -TERM $$`...eh, but `kill -TERM $$ && sleep infinity` is obviously 50% worse or something)